src/Microsoft.AspNet.Mvc.WebApiCompatShim/ApiController.cs

```diff
@@ -160,8 +168,13 @@ public virtual ConflictResult Conflict()
         /// <param name="value">The content value to negotiate and format in the entity body.</param>
         /// <returns>A <see cref="NegotiatedContentResult{T}"/> with the specified values.</returns>
         [NonAction]
-        public virtual NegotiatedContentResult<T> Content<T>(HttpStatusCode statusCode, [NotNull] T value)
+        public virtual NegotiatedContentResult<T> Content<T>(HttpStatusCode statusCode, T value)
         {
+            if (value == null)
```

Comment:
This surprisingly doesn't cause a compilation issue and works at runtime for value types.